### DISCUSSION TITLE
Welcome to CodeIT Discussions!

### DISCUSSION THREAD

**Discussion by victor-pogor** [victor-pogor/CodeIT#51] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sosiristseng** [comment]
Hello, a previous LoveIt theme user and a computational biology Ph.D. student here. Thank you for taking up the torch to revive this beautiful, performant, and functional theme.

Currently, I set up a [template repo](https://github.com/sosiristseng/template-hugo-codeit) for testing and future use. I'm starting to migrate [my notes from a Hexo blog](https://sosiristseng.github.io/) to this when I get some free time.
